src/libraries/System.Text.Json/tests/System.Text.Json.Tests/TrimmingTests/System.Text.Json.TrimmingTests.proj

```diff
@@ -1,7 +1,9 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
-  <ItemGroup>
+  <!-- These tests need to be rewritten to use source generator: https://github.com/dotnet/runtime/issues/53437,
```

Comment:
This issue is closed. Can we either reopen it or open a new one?

Reply:
Reopened #53437 but per my experience, something in Future milestone is usually as good as closed